src/components/containers/BooksForm.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/forbid-prop-types */
 import React, { Component } from 'react';
```

Comment:
Do not disable the eslint rule. Kindly remove the disable comment on line 1 and fix the issue

Reply:
But that one can't be fixed, since it is  mandatory in some cases to pass object or array in props